### ISSUE TITLE
Rule MD004 incorrectly flagging on an Ordered List

### ISSUE THREAD

**Issue by gryte** [DavidAnson/markdownlint#11]
I'm getting flagged on the following. Am I missing something obvious?

```
1. Fork the repository on Github
+ Create a named feature branch (like `add_component_x`)
+ Write your change
+ Write tests for your change (if applicable)
+ Run the tests, ensuring they all pass
+ Submit a Pull Request using Github
```

**Comment by DavidAnson** [comment]
Are you on the latest version of markdownlint?

I pasted your example into the top-left box of http://dlaa.me/markdownlint/ and do not see any MD004 violations.

Could you please try that?

**Comment by gryte** [comment]
Interesting...yeah, I see what you mean if I use the website you referenced (nice by they way).

I confirmed that I'm on markdownlint version 0.3.0 which seems to be the latest in the marketplace.

I feel like I'm not using the correct md syntax to make an ordered list, but when I try the following, I get MD029 in both code and the website.

```
1. Fork the repository on Github
2. Create a named feature branch (like `add_component_x`)
3. Write your change
4. Write tests for your change (if applicable)
5. Run the tests, ensuring they all pass
6. Submit a Pull Request using Github

```

Do you have a good reference for appropriate syntax handy?

Thanks for the help!

**Comment by DavidAnson** [comment]
You saying that version 0.3.0 is the latest in the marketplace makes me think you're referring to the markdownlint VS Code extension https://github.com/DavidAnson/vscode-markdownlint (which uses this markdownlint engine). However, I've tried your original example with that and still do not see the error you're reporting.

The reason your second example triggers MD029 is because the default configuration for MD029 is to prefer the style 1./1./1. over 1./2./3. because the latter is easier to get wrong. This is discussed in the relevant documentation page: https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md029---ordered-list-item-prefix You can change the default in the usual manner - here's an example: https://github.com/DavidAnson/markdownlint/blob/master/test/ordered_list_item_prefix_ordered.json For details on making that change for vscode-markdownlint, please see: https://github.com/DavidAnson/vscode-markdownlint#configure

Regarding documentation about syntax, I like the following:
* http://daringfireball.net/projects/markdown/
* http://commonmark.org/

**Comment by gryte** [comment]
Sorry I failed to mention I was in vs code. Thanks for the clarification!

**Comment by DavidAnson** [comment]
Closing this issue because it seems to have been resolved. If you're still having difficulty, let me know!

PS - I recently updated the VS Code markdownlint extension to v0.4.1, so please consider upgrading. :)
